packages/js/src/MatomoTracker.test.ts

```diff
@@ -3,6 +3,18 @@ import MatomoTracker from './MatomoTracker'
 const URL_BASE = 'https://example.com'
 
 describe('MatomoTracker', () => {
+  it('throws an error if no urlBase is passed in options', () => {
+    // eslint-disable-next-line
+    // @ts-expect-error
+    expect(() => new MatomoTracker({ siteId: 1 })).toThrowError()
+  })
+
+  it('throws an error if no siteId is passed in options', () => {
+    // eslint-disable-next-line
+    // @ts-expect-error
+    expect(() => new MatomoTracker({ urlBase: URL_BASE })).toThrowError()
+  })
```

Comment:
Instead of adding these comments to disable ESLint and the TypeScript compiler you can use `as UserOptions` to force the argument to be accepted.

```suggestion
  it('throws an error if no urlBase is passed in options', () => {
    expect(() => new MatomoTracker({ siteId: 1 } as UserOptions)).toThrowError()
  })

  it('throws an error if no siteId is passed in options', () => {
    expect(() => new MatomoTracker({ urlBase: URL_BASE } as UserOptions)).toThrowError()
  })
```

Reply:
Good call. Done.